test/integration/LBQuoterPriority.t.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.10;
+
+import "../helpers/TestHelper.sol";
+
+/**
+ * Market deployed:
+ * - USDT/USDC, V1 with low liquidity, V2 with high liquidity
+ * - WNATIVE/USDC, V1 with high liquidity, V2 with low liquidity
+ * - WETH/USDC, V1 with low liquidity, V2.1 with high liquidity
+ * - BNB/USDC, V2 with high liquidity, V2.1 with low liquidity
+ *
+ * Every market with low liquidity has a slighly higher price.
+ * It should be picked with small amounts but not with large amounts.
+ * All tokens are considered 18 decimals for simplification purposes.
+ */
```

Comment:
Can you specify what's tested here ? Will be confusing if we add more updates on top of this one